heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -48,15 +50,13 @@
 import com.spotify.heroic.querylogging.QueryLoggerFactory;
 import com.spotify.heroic.statistics.DataInMemoryReporter;
 import com.spotify.heroic.statistics.MetricBackendReporter;
+
```

Comment:
Reladed formatter from master and no more blank lines. Should be changed in all files, right? Will cause some merging down the line.